scripts/webui.py

```diff
@@ -773,7 +773,15 @@ def classToArrays( items, seed, n_iter ):
 
     return all_seeds, n_iter, prompt_matrix_parts, all_prompts, needrows
 
-
+def replace_wildcard(chunk):
+    if not " " in chunk:
+        file_dir = os.path.dirname(os.path.realpath("__file__"))
+        replacement_file = os.path.join(file_dir, "scripts/wildcards/" + chunk + ".txt")
+        if os.path.exists(replacement_file):
+            lines = open(replacement_file, encoding="utf8").read().splitlines()
+            my_line = random.choice(lines)
+            return(my_line)
+    return(chunk)
```

Comment:
Using a `with` block ensures the file is closed in a timely fashion.
```suggestion
            with open(replacement_file, encoding="utf8") as f:
                return random.choice(f.read().splitlines())
    return chunk
```